### PR TITLE
OPENEUROPA-2693: On IE11 the icons are not displayed.

### PR DESCRIPTION
**Pull request by sergepavle**
### Description

[Insert description here]

### Change log

- Added:
- Changed:
- Deprecated:
- Removed:
- Fixed:
- Security:

### Commands

```sh
[Insert commands here]

```

### CODE REVIEW

**Review by 22Alexandra — DISMISSED** [review]

File: js/svg4ie.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/**
+ * @file
+ * ECL fix for SVGs in IE 11.
+ */
+(function (ECL, Drupal) {
```
Comment (brummbar) [inline js/svg4ie.js]: No need to have ECL as parameter.

File: js/svg4ie.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  Drupal.behaviors.svg4Ie = {
+    attach: function attach() {
+      if (typeof svg4everybody === "function" && !!navigator.userAgent.match(/Trident.*rv\:11\./)) {
```
Comment (brummbar) [inline js/svg4ie.js]: No need to test that `svg4everybody` is defined, we already set the dependency.
Also no need to test for browser, the library does it already.

File: oe_theme.libraries.yml [hunk 3]
```diff
@@ -6,8 +6,10 @@ component_library_ec:
   js:
     dist/ec/scripts/ecl-ec-preset-legacy-website.js: {}
     js/ecl_auto_init.js: {}
+    js/svg4ie.js: { preprocess: false }
```
Comment (brummbar) [inline oe_theme.libraries.yml]: Why disabling the aggregation on this script?

File: oe_theme.libraries.yml [hunk 4]
```diff
@@ -17,8 +19,10 @@ component_library_eu:
   js:
     dist/eu/scripts/ecl-eu-preset-legacy-website.js: {}
     js/ecl_auto_init.js: {}
+    js/svg4ie.js: { preprocess: false }
```
Comment (brummbar) [inline oe_theme.libraries.yml]: Same as above.

File: oe_theme.libraries.yml [hunk 5]
```diff
@@ -30,3 +34,7 @@ dialogs:
     js/dialogs.js: {}
   dependencies:
     - core/drupal
+
+svg4everybody:
```
Comment (brummbar) [inline oe_theme.libraries.yml]: To know which version of the library we included, let's use the version here.

File: oe_theme.libraries.yml [hunk 6]
```diff
@@ -30,3 +34,7 @@ dialogs:
...
+  js:
+    js/svg4everybody.min.js: { preprocess: false }
```
Comment (brummbar) [inline oe_theme.libraries.yml]: Also here, why disabling aggregation?

**Review by brummbar — CHANGES_REQUESTED** [review]
Note from @ademarco: let's add the npm package `svg4everybody` as dependency, and then use the ecl-builder to copy it over in the dist package instead of putting the library file here.

File: oe_theme.libraries.yml (hunk 5 above)
Comment (brummbar) [reply oe_theme.libraries.yml]: This is not needed anymore, as we will use npm to handle the library.

File: js/svg4ie.js (hunk 2 above)
Comment (brummbar) [reply js/svg4ie.js]: Since we won't be testing for browser, let's remove the `IE` stuff. It's just `svg4Everybody` the behavior name now.

**Review by ademarco — CHANGES_REQUESTED** [review]
I'd declare one single library as such:
```yml
external_svg:
  js:
    dist/js/svg4everybody.min.js: { minified: true }
    js/external_svg.js: {}
```
And then we can simply load this library via the hteme info.yml file, as we do with the `editor` library:

```yml
libraries:
  - oe_theme/editor
  - oe_theme/external_svg
```

File: oe_theme.libraries.yml (hunk 3 above)
Comment (sergepavle) [reply oe_theme.libraries.yml]: Left over on try to fix order of js files.
Removed.

File: js/external_svg.js [hunk 7]
```diff
@@ -0,0 +1,11 @@
+/**
+ * @file
+ * ECL fix for SVGs in IE 11.
+ */
+(function (Drupal) {
+  Drupal.behaviors.svg4Everybody = {
+    attach: function attach() {
```
Comment (brummbar) [inline js/external_svg.js]: Missed this last time: no need to name the function.
```suggestion
    attach: function () {
```

File: ecl-builder.config.js [hunk 8]
```diff
@@ -63,5 +63,6 @@ module.exports = {
     { from: path.resolve(nodeModules, '@ecl-twig'), patterns: 'ec-component-tag/*.twig', to: path.resolve(outputFolder, 'components') },
     { from: path.resolve(nodeModules, '@ecl-twig'), patterns: 'ec-component-text-input/*.twig', to: path.resolve(outputFolder, 'components') },
     { from: path.resolve(nodeModules, '@ecl-twig'), patterns: 'ec-component-timeline/*.twig', to: path.resolve(outputFolder, 'components') },
+    { from: path.resolve(nodeModules, 'svg4everybody/dist'), to: path.resolve(outputFolder, 'dist/js') },
```
Comment (brummbar) [inline ecl-builder.config.js]: Let's copy only the minified version of the script.

File: oe_theme.libraries.yml [hunk 9]
```diff
@@ -30,3 +30,8 @@ dialogs:
     js/dialogs.js: {}
   dependencies:
     - core/drupal
+
+external_svg:
+  js:
+    dist/js/svg4everybody.js: {}
```
Comment (brummbar) [inline oe_theme.libraries.yml]: So here we put the minified version.

**Review by brummbar — CHANGES_REQUESTED** [review]

**Review by brummbar — APPROVED** [review]

**Review by ademarco — APPROVED** [review]

**Pull request merged**